auslib/test/admin/views/test_rules.py

```diff
@@ -1206,10 +1208,11 @@ def testGetScheduledChangesWithCompleted(self):
         self.assertEquals(json.loads(ret.data), expected)
 
     def testAddScheduledChangeExistingRule(self):
+        future_time = int(time.mktime((datetime.datetime.now() + datetime.timedelta(minutes=5)).timetuple()) * 1000)
```

Comment:
This works, but please following the existing pattern for this by using mock.patch instead, like: https://github.com/andela/balrog/blob/20ee523439c183b9581812f23afe2a69f5d2b706/auslib/test/admin/views/test_rules.py#L1289.